tests/collections/options.test.ts

```diff
@@ -128,7 +128,7 @@ describe('Options tests', async () => {
             //rawResult options should be cleaned up by stargate-mongoose, but 'upsert' should be preserved
             const updateOneResp = await Product.updateOne({ name: 'Product 4' },
                 { $set : { isCertified : true }, $inc: { price: 5 } },
-                { upsert: true, rawResult: false, sort: { name : 1 } }
+                { upsert: true, rawResult: false, sort: { name : 1 } } as unknown as Record<string, never>
```

Comment:
@vkarpov15  I'm sorry could you please explain why `as unknown as Record<string, never>` is required here now?

Reply:
TypeScript complains that `rawResult` is not an allowed property. Mongoose 8.1 made the `options` parameter for `Product.updateOne()` a bit more strict: Mongoose now disallows any options for `Product.updateOne()` that aren't supported by `updateOne()`, including `rawResult` and `sort`.